frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/general-tab.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Grid } from '@patternfly/react-core';
+import { iGetVmSettings } from '../../selectors/immutable/vm-settings';
+import { VMSettingsField, VMWizardProps } from '../../types';
+import { FormFieldReviewMemoRow } from '../../form/form-field-review-row';
+import { FormFieldType } from '../../form/form-field';
+import { iGetCommonData, iGetLoadedCommonData } from '../../selectors/immutable/selectors';
+import { getOS } from '../../../../k8s/requests/vm/create/common';
+import { VMSettings } from '../../redux/initial-state/types';
+import { getField } from './utils';
+import { FlavorReviewRow } from './FlavorRow';
+
+import './general-tab.scss';
+
+const GeneralReviewConnected: React.FC<GeneralReviewConnectedProps> = (props) => {
+  const { vmSettings, iUserTemplates, iCommonTemplates, openshiftFlag } = props;
+
+  const osName =
```

Comment:
recently `osField.display` property was added - which is filled only when `osField.value` is missing and id is unknown. 

`osField.display` is used only by ovirt provider which does not require os id

Reply:
I apologize. I thought I understood what needed to be done here after our conversation yesterday, but apparently I don't. Could you please clarify?

Reply:
we should show the osField.display if osField.value is missing (the ID)

Reply:
Done!